src/util.ts

```diff
@@ -24,8 +24,14 @@ const REGION_STEP_RANGES = {
   CN: {
     start: 14,
     end: 30,
-    step:0.5
-  }
+    step: 0.5,
+  },
+  // TODO: verify the Australian temp code ranges
```

Comment:
I'm like 90% sure this is correct, but will need to validate more in the future with some other vehicles just in case.